docs/stencil-docs/reference-docs/theme-objects.md

```diff
@@ -69,15 +69,14 @@ To use dynamic data from the template in client-side code, use the inject helper
 
 <!-- ... -->
 
-<script>
-    var jsContext = JSON.parse({{jsContext}});
-</script>
+var jsContext = JSON.parse({{jsContext}});
```

Comment:
This line (the entire <script> tag and its contents) doesn't need to be in this example.